app/src/main/java/org/gdg/frisbee/android/adapter/ChapterAdapter.java

```diff
@@ -76,4 +76,14 @@ public long getItemId(int position) {
         else
             return 0;
     }
+
+    public Chapter findById(final String chapterId) {
+        for (int i = 0; i < getCount(); i++) {
```

Comment:
I was thinking about this. 
Since `equals` override in Chapter uses GplusId, we can use `indexOf` function in the ArrayList stored in this adapter.

I would be more elegant? Maybe not? What do you think?